packs/maven-java16/.lighthouse/jenkins-x/pullrequest.yaml

```diff
@@ -14,7 +14,7 @@ spec:
           env:
           - name: MAVEN_OPTS
             value: -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=25 -Dmaven.wagon.http.retryHandler.count=3
-          image: uses:jenkins-x/jx3-pipeline-catalog/tasks/maven-java14/pullrequest.yaml@versionStream
+          image: uses:jenkins-x/jx3-pipeline-catalog/tasks/maven-java16/pullrequest.yaml@versionStream
```

Comment:
Should this not be 17?

Reply:
Never mind, I see now ...

Reply:
yes,  I noticed this was wrong incidentally.